hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/log/LogLevel.java

```diff
@@ -51,6 +47,7 @@
 import org.apache.hadoop.util.ServletUtil;
 import org.apache.hadoop.util.Tool;
 import org.apache.hadoop.util.ToolRunner;
+import org.apache.log4j.Logger;
```

Comment:
with this as an explicit import, `process(org.apache.log4j.Logger log, ...)` should remove its fully qualified import

Reply:
done